SConstruct

```diff
@@ -538,6 +538,13 @@ o.Add(
 	"/usr/local/appleseed/lib",
 )
 
+# compatibility options
+
+o.Add(
+	BoolVariable( "WITH_CORTEX10_COMPAT", "Set this to include cortex 10 compatibility modules.", True ),
```

Comment:
As suggested, added the `WITH_CORTEX10_COMPAT` variable.
I've set it as default to `True`, since I don't see a significant drawback in installing it.

An alternative would be to set it to `False` and then enable it by default in the `ie/options` file.
But we'd still need to remember to enable it when doing local development installs.

Reply:
>  But we'd still need to remember to enable it when doing local development installs.

The IE config file still takes effect for dev builds doesn't it?

Reply:
My preference was for the default to be `False`, although I don't really have a strong case for it - whatever you think best.